lessonbuilder/tool/src/webapp/templates/ShowPage.html

```diff
@@ -2424,7 +2424,7 @@ <h3><span rsf:id="msg=simplepage.layout.add.from.template"></span></h3>
                                         <div class="form-group">
                                             <h4><span rsf:id="msg=simplepage.layout.section.title"></span></h4>
                                             <label class="sr-only" for="layout-section-title"><span rsf:id="msg=simplepage.layout.section.title"></span></label>
-                                            <input type="text" class="form-control" id="layout-section-title" rsf:id="layout-section-title" placeholder="Section Title" />
+                                            <input type="text" class="form-control" id="layout-section-title" rsf:id="layout-section-title" />
```

Comment:
Instead of removing it outright, can you provide a msg bundle with translation and access it here?

```suggestion
                                            <input type="text" class="form-control" id="layout-section-title" rsf:id="layout-section-title" placeholder="msg=msg.bundle.key" />
```

Reply:
@bjones86 I tested placeholder="msg=msg.bundle.key" and it didn't work so I decide to remove the placeholder.
There isn't any working example with the placeholder (at least on lessons).

Reply:
Thanks for trying. Maybe others have an idea? You could use JavaScript to add it after the fact, but I'm not a huge fan of that approach.

Reply:
@jesusmmp did you look in the ShowPageProducer to see if it is being set there?

Reply:
@bjones86, @ern . Changes are done!